### PR TITLE
[Editor] Avoid editor creation/selection on right click (bug 1781762)

### PR DESCRIPTION
**Pull request by calixteman**

### CODE REVIEW

**Comment by calixteman** [comment]
Unfortunately, I didn't manage to write a test for that because I was unable to dismiss correctly (without any mouse/keyboard actions) the context menu.

File: src/display/editor/annotation_editor_layer.js [hunk 1]
```diff
@@ -577,6 +577,16 @@ class AnnotationEditorLayer {
...
+    const isMac = KeyboardManager.platform.isMac;
+    if (event.button !== 0 || (event.ctrlKey && isMac)) {
+      // Don't create an editor on right click.
+      return;
+    }
+
+    if (event.button !== 0) {
+      return;
+    }
+
```
Comment (Snuffleupagus) [inline src/display/editor/annotation_editor_layer.js:589]: Isn't this block completely redundant, given that you already return early above when this condition is satisfied?

**Review by Snuffleupagus — APPROVED** [review]

**Comment by calixteman** [comment]
The failure on the windows bot is a bit crazy...
```
In firefox: Expected 'Hello PDF.js Worl' to equal 'Hello PDF.js World !!'.
```

**Pull request merged**